apps/web/components/ActionButtons.tsx

```diff
@@ -114,6 +133,31 @@ export function ActionButtons(props: {
           <ArchiveIcon className="h-5 w-5 text-gray-700" aria-hidden="true" />
         ),
       },
+      isStateRead
+        ? {
+            tooltip: "Mark Unread",
+            onClick: onReadUnread,
+            icon: isReadUnreading ? (
+              <LoadingMiniSpinner />
+            ) : (
+              <MailMinusIcon
+                className="h-5 w-5 text-gray-700"
+                aria-hidden="true"
+              />
+            ),
+          }
+        : {
+            tooltip: "Mark Read",
+            onClick: onReadUnread,
+            icon: isReadUnreading ? (
+              <LoadingMiniSpinner />
+            ) : (
+              <MailCheckIcon
+                className="h-5 w-5 text-gray-700"
+                aria-hidden="true"
+              />
+            ),
+          },
```

Comment:
Gmail uses the terminology "Mark as read" and "Mark as unread" - I'd use that so you don't have to reinvent the wheel here.